docs/modules/ROOT/pages/guide-keep3r.adoc

```diff
@@ -0,0 +1,138 @@
+____
+
+[[prerequsites]]
+## Prerequisites
+
+To get started, you'll need the following:
+
+. *A Defender mainnet account*. Head over to https://defender.openzeppelin.com/[Defender] to sign up for a new account, and then https://openzeppelin.com/apply/[apply for mainnet access] indicating that you are interested in running a keep3r. We'll contact you shortly to activate your account for mainnet access. Note that mainnet accounts are free for now, but will be paid starting end of year.
+
+. *ETH to pay for transactions gas*. Keeper transactions require gas for execution, so make sure to have some ETH available to fund your Relay.
+
+. *An Alchemy, Etherscan, and/or Infura key (optional)*. Running a Keeper requires querying which jobs are available, and https://dashboard.alchemyapi.io/signup?referral=53fcee38-b894-4d5f-bd65-885d241f8d29[Alchemy], https://infura.io/[Infura], and https://etherscan.io/apis[Etherscan] provide free access to the network for executing these queries. You can do without them, but you may get throttled during your Keepers execution. Read more about this https://docs.ethers.io/v5/api-keys/[here].
+
+## Creating a Relayer
+
+The first step will be to create a Defender Relayer to act as your Keeper. This Relayer will be the Ethereum account you will be executing jobs from. 
+
+To create a Relayer, simple head over to the Relayer section in Defender, and create a new one on the Mainnet network.
+
+WARNING: If the option to create the relayer in mainnet is locked, it means your Defender account does not have mainnet access yet. Make sure to https://openzeppelin.com/apply/[apply for mainnet access first] and wait for our response.
+
+## Fund your Relayer
+
+Next step is to send ETH to your Relayer, so it has funds to execute the jobs. Copy the Ethereum address of your new Relayer, and use your favorite wallet to send ETH to it. You can send whatever amount you want, but you want at least around 0.2 ETH to be able to send a handful of transactions before running out.
+
+NOTE: Defender will send you an email when your Relay's funds drop below 0.1 ETH to remind you to refill it.
+
+## Create an Autotask
+
+Now that you have your Relay ready to send transactions, let's set up the Defender Autotask that will instruct the Relay to send them. An Autotask is a snippet of code that runs at regular intervals, in this case, we'll use it to automate the process of bonding, activating, searching for jobs, and executing them.
+
+Create a new Autotask in the Autotasks section of Defender, choosing to run every few minutes, and connecting it to the Relay you had created previously.
+
```

Comment:
We should add an explanation that the code below registers a Keeper and once activated will attempt to do work for the `YearnV1EarnKeep3r` as an example.